src/Mobs/Cow.h

```diff
@@ -20,7 +20,10 @@ class cCow :
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	virtual void OnRightClicked(cPlayer & a_Player) override;
 
-	virtual const cItem GetFollowedItem(void) const override { return cItem(E_ITEM_WHEAT); }
+	virtual void GetFollowedItems(cItems & a_Items) override
```

Comment:
Not saying it wouldn't work, but wouldn't it need to call the copy-constructor on exit to create a new copy of the same array every time? Gets even worse once you start having more overrides underneath it.